wagtail/contrib/search_promotions/forms.py

```diff
@@ -7,6 +7,14 @@
 
 
 class QueryForm(forms.Form):
+    """
+    Query form for search promotions
+
+    Attributes
+
+        - query_string (forms.CharField): Query text to match.
+    """
```

Comment:
I find those docstrings a bit distracting, and in this instance I can’t see any information in the comment that’s not already clear from the code. If this is for existing models where nothing changes, could we either not do this or do this in a separate PR? What’s the purpose here?

Similarly for method-level docstrings, they don’t seem to add much so I’d recommend removing them all (or if we think some of them are valuable, only add them to existing code in a separate PR).

Reply:
@TopDevPros this concern is still not addressed, adding docstrings to unrelated code changes is very distracting from this PR and we cannot merge until these are removed.

I understand your desire to 'clean things' or 'make things consistent', I too struggle with that.

However, our goal is a single (or small set) of commits that make one self-contained change. Changing these docstrings here is not required for this feature.